integration-cli/requirements.go

```diff
@@ -32,6 +33,13 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	Network = TestRequirement{
+		func() bool {
+			_, err := http.Get("http://google.com")
```

Comment:
Yep, When you said no env variables, I thought this is one of the ways to detect network availability, I am open to suggestions.  I can also also use ping onto Docker hub?